.gitignore

```diff
@@ -36,4 +36,9 @@ AssemblyInfo.cs
 hibernate.cfg.xml
 available-test-configurations
 current-test-configuration
-NHibernate.VisualState.xml
\ No newline at end of file
+NHibernate.VisualState.xml
+/src/packages/*/
+/Examples/EntityValidator/NHibernate.Validator.Demo.Ev/packages/*/
+/Examples/Integration/NHibernate.Validator.Demo.NHI/packages/*/
+/Examples/IoC/NHibernate.Validator.Demo.IoC.Windsor/packages/*/
+/Examples/Winforms/NHibernate.Validator.Demo.Winforms/packages/*/
```

Comment:
Other NHibernate project use rule like following one for this NuGet:

```
# NuGet Packages
*.nupkg
# The packages folder can be ignored because of Package Restore
**/packages/*
# except build/, which is used as an MSBuild target.
!**/packages/build/
# If using the old MSBuild-Integrated Package Restore, uncomment this:
#!**/packages/repositories.config
```

If updating the gitignore, better also and rules for new tooling:
```
# Visual Studio user settings directory
.vs

# Visual Studio Code user settings directory
.vscode

# JetBrains Rider IDE user settings directory
.idea
```

Reply:
Done - why not
But in nhibernate-core it is simple:
packages/
and nothing about 
*.nupkg
!**/packages/build/
Why?

Reply:
NHibernate-core does not have multiple solutions in distinct folders, that simple rule directly in src folder is enough.